tests/mock_loader.py

```diff
@@ -48,3 +48,25 @@ def get_dataset_object(self, ref):
             raise RuntimeError('dataset ref not found: "{}"'.format(ref))
         return MOCK_DATASET_LIST[index]
 
+class DatasetMockLoader(object): # TODO - better name and place
```

Comment:
As for the name of this class, it's not a huge deal, and the code was already playing a bit fast and loose with the standard definition of "mock", which a bit different from how it's used in out in the wild.

That being said, it is a problem that this name is very similar to the other `MockLoader` while they're doing very different things. The interesting thing about this one is that it is given the data it returns. How about `ResponseLoader` or `SettableLoader` or something like that?